diff --git a/packages/lib/utils/processStartFlags.ts b/packages/lib/utils/processStartFlags.ts
index 9752851..6869d58 100644
--- a/packages/lib/utils/processStartFlags.ts
+++ b/packages/lib/utils/processStartFlags.ts
@@ -160,6 +160,13 @@ const processStartFlags = async (argv: string[], setDefaults = true) => {
 			continue;
 		}
 
+    if (arg === '--enable-wayland-ime') {
+      // Electron-specific flag - ignore it
+      // Allows users to enable wayland ime
+      argv.splice(0, 1);
+      continue;
+    }
+
 		if (arg === '--disable-smooth-scrolling') {
 			// Electron-specific flag - ignore it
 			// Allows users to disable smooth scrolling
