diff --git a/package.json b/package.json
index 694d34f0e..c19e35fd3 100644
--- a/package.json
+++ b/package.json
@@ -109,5 +109,6 @@
     "pdfjs-dist": "patch:pdfjs-dist@npm%3A3.11.174#./.yarn/patches/pdfjs-dist-npm-3.11.174-67f2fee6d6.patch",
     "@react-native-community/slider": "patch:@react-native-community/slider@npm%3A4.4.4#./.yarn/patches/@react-native-community-slider-npm-4.4.4-d78e472f48.patch",
     "husky": "patch:husky@npm%3A3.1.0#./.yarn/patches/husky-npm-3.1.0-5cc13e4e34.patch"
+    "chokidar@^2.0.0": "3.5.3"
   }
 }
diff --git a/packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/useContextMenu.ts b/packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/useContextMenu.ts
index 6289d8419..580df524d 100644
--- a/packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/useContextMenu.ts
+++ b/packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/utils/useContextMenu.ts
@@ -1,5 +1,5 @@
 
-import { ContextMenuEvent, ContextMenuParams } from 'electron';
+import { ContextMenuParams, Event } from 'electron';
 import { useEffect, RefObject } from 'react';
 import { _ } from '@joplin/lib/locale';
 import { PluginStates } from '@joplin/lib/services/plugins/reducer';
@@ -69,7 +69,7 @@ const useContextMenu = (props: ContextMenuProps) => {
 			return intersectingElement && isAncestorOfCodeMirrorEditor(intersectingElement);
 		}
 
-		async function onContextMenu(event: ContextMenuEvent, params: ContextMenuParams) {
+		async function onContextMenu(event: Event, params: ContextMenuParams) {
 			if (!pointerInsideEditor(params)) return;
 
 			// Don't show the default menu.
diff --git a/packages/app-desktop/package.json b/packages/app-desktop/package.json
index 9a9682758..cc878dcc9 100644
--- a/packages/app-desktop/package.json
+++ b/packages/app-desktop/package.json
@@ -121,7 +121,7 @@
     "@electron/rebuild": "3.3.0",
     "@joplin/default-plugins": "~2.14",
     "@joplin/tools": "~2.14",
-    "@playwright/test": "1.40.1",
+    "@playwright/test": "1.42.1",
     "@testing-library/react-hooks": "8.0.1",
     "@types/jest": "29.5.8",
     "@types/node": "18.19.8",
@@ -129,8 +129,8 @@
     "@types/react-redux": "7.1.33",
     "@types/styled-components": "5.1.32",
     "@types/tesseract.js": "2.0.0",
-    "electron": "26.5.0",
-    "electron-builder": "24.8.0",
+    "electron": "29.1.0",
+    "electron-builder": "24.13.3",
     "glob": "10.3.10",
     "gulp": "4.0.2",
     "jest": "29.7.0",
@@ -143,7 +143,7 @@
   },
   "dependencies": {
     "@electron/notarize": "2.1.0",
-    "@electron/remote": "2.0.12",
+    "@electron/remote": "2.1.2",
     "@fortawesome/fontawesome-free": "5.15.4",
     "@joeattardi/emoji-button": "4.6.4",
     "@joplin/editor": "~2.14",
diff --git a/packages/app-desktop/tools/electronRebuild.js b/packages/app-desktop/tools/electronRebuild.js
index 4864d507f..d4cd7ee6e 100644
--- a/packages/app-desktop/tools/electronRebuild.js
+++ b/packages/app-desktop/tools/electronRebuild.js
@@ -25,7 +25,7 @@ async function main() {
 	// wrong one. However it means it will have to be manually upgraded for each
 	// new Electron release. Some ABI map there:
 	// https://github.com/electron/node-abi/tree/master/test
-	const forceAbiArgs = '--force-abi 116';
+	const forceAbiArgs = '--force-abi 122';
 
 	if (isWindows()) {
 		// Cannot run this in parallel, or the 64-bit version might end up
